lib/node_modules/@stdlib/math/base/special/negafibonacci/benchmark/benchmark.js

```diff
@@ -46,10 +46,11 @@ bench( pkg, function benchmark( b ) {
 	var y;
 	var i;
 
+	x = -discreteUniform( 100, 0, 78 );
```

Comment:
This doesn't work. `discreteUniform( 100, 0, 78 );` returns an array and we cannot just stick a `-` in front to get negative values. Instead, please directly draw from the desired distribution, i.e. do `discreteUniform( 100, -78, 0 );` here.

Similar changes will have to be made in various other places of this PR.